cmd/ebrelayer/relayer/ethereum.go

```diff
@@ -161,9 +160,12 @@ func (sub EthereumSub) Start(completionEvent *sync.WaitGroup) {
 	signal.Notify(quit, syscall.SIGINT, syscall.SIGTERM)
 	defer close(quit)
 
-	// Start BridgeBank subscription, prepare contract ABI and LockLog event signature
-	bridgeBankAddress, subBridgeBank := sub.startContractEventSub(logs, client, txs.BridgeBank)
-	defer subBridgeBank.Unsubscribe()
+	// get the bridgebank address from the registry contract
+	bridgeBankAddress, err := txs.GetAddressFromBridgeRegistry(client, sub.RegistryContractAddress, txs.BridgeBank)
+	if err != nil {
```

Comment:
Shouldn't this be fatal?  If we can't get the bridgeBankAddress, we'll still try to use it later on.

Reply:
Agree with James, we can exit here if not get the bridge bank address.

Reply:
Added fatal if bridgebank address is not found.

Reply:
FYI, that's just marking the log message as fatal, it doesn't exit.